src/components/PatternPlayground/PatternPlayground.tsx

```diff
@@ -28,6 +22,7 @@ const PATTERN_PREVIEW_DISPLAY_SIZE = 600;
 export const PatternPlayground = observer(function PatternPlayground() {
   const store = useStore();
   const { uiStore } = store;
+  const initIndices : number[] = [];
```

Comment:
When I opened this file, `prettier` made some whitespace changes, like deleting the space before the colon on this line, just FYI. Do you use prettier? Maybe we should have a .prettierrc file if so to make sure our settings are aligned.

Reply:
My VSCode is actually pretty bare bones! I'll look into adding `prettier`. If you have any extensions you recommend, I'll gladly take a look.

Reply:
Ok cool, Prettier and ESLint I would recommend. I think if you use those then we will both be seeing the same squiggly error lines and can work on keeping warnings/errors to a minimum.